projects/plugins/crm/src/automation/class-attribute-definition.php

```diff
@@ -75,14 +67,6 @@ class Attribute_Definition {
 	 */
 	const NUMBER = 'number';
 
-	/**
-	 * Represents a password input.
-	 *
-	 * @since $$next-version$$
-	 * @var string
-	 */
-	const PASSWORD = 'password';
```

Comment:
Why is password removed? I wanted to remove it but there were arguments made in favor of having it.

Reply:
My reasoning is: It's fair enough if there is a use-case for it in the future that we add it then, but handling password needs encrypting etc as well, so IMO we shouldn't mess with it now that we're aiming for feature parity with Automations v1 (and we're already doing more than that).

Do you remember where the discussion was? More so I don't miss anything for feature parity.

Reply:
The discussion was just that certain third party systems might require passwords. I'm fine merging without that though.